src/layout/generic-input/utils/generic-input-utils.spec.js

```diff
@@ -195,7 +195,7 @@ describe('getNewLoopPlaceholder', () => {
 
     };
     const result = getNewLoopPlaceholder(components);
-    expect(result).toEqual(true);
+    expect(result).toEqual(undefined);
```

Comment:
If you want to convert to a boolean value, you can two extra !! to the implementation. 

return !!Object.values(components).find(component => component.type === SEQUENCE);